src/views/Game/Game.tsx

```diff
@@ -2629,7 +2629,7 @@ export class Game extends React.PureComponent<GameProperties, GameState> {
                 }
                 {(this.state.phase === "finished" || null) &&  /* { */
                     <div className="analyze-mode-buttons">     {/* not really analyze mode, but equivalent button position and look*/}
-                        {(this.state.user_is_player && this.state.mode !== "score estimation" || null) &&
+                        {(this.state.user_is_player && this.state.mode !== "score estimation" && !this.goban.engine.rengo || null) &&
```

Comment:
Why is it that people who have just completed a rengo game are less likely to want a rematch than those who complete a regular game? 

Or was the code just not playing nice and this feature is out of scope of an initial implementation?

Reply:
To quote anoek:

"
* When sending a challenge to someone, either the rengo box should be hidden, or we should do something like navigate to the match making screen, add additional invites, or something of the sort. Currently, it just hangs even if the other player accepts.
"

... hence... ;)

Reply:
Thanks, I missed that. Makes sense.

Reply:
Actaully, this is the anoek comment I was looking for, but similar sentiment:

"
* When the game is over all or none of players should see the "Rematch" button". Ideally "All", but it seems like that might be a bit challenging with rengo and expecatations and matchmaking, so maybe just hide it.
"

Reply:
Yeah to be clear, I have no problem with a *functional* rematch button, but the current rematch flow would need a good bit of work to make that work, and I didn't see it as critical to Rengo so I didn't want it to block merging.